### ISSUE TITLE
Amazon Prime video doesn't seem to work consistently

### ISSUE THREAD

**Issue by RobertYost** [dvlden/ultrawideo#18]
### Describe the bug
For some reason the extension doesn't work consistently on amazon prime video consistently. At first, the extension wasn't working, no matter what I tried, and then suddenly it started working, and now it's not working anymore again. I have not made any changes to the browser outside of the extension during this period.

### Reproduction steps
1. Watch video on amazon prime video

### Expected behavior
I expected it to work consistently
### Environment
- **OS**: Windows 10.0 Build 18362
- **Browser**: Firefox 68.0.1 and Chrome 76.0.3809.100

**Comment by dvlden** [comment]
Thank you for using the template in both issues. I appreciate that.
I'll take a look at this issue as soon as possible and I'll address it. I must get Amazon Prime Video first...

If you don't mind sharing your credentials for testing purposes (one time), let me know and I'll give you my e-mail address.

**Comment by StealthyApple** [comment]
Hello Dvlden, I was wondering if you still needed access to Amazon Prime in order to resolve this issue. I am new to the extension and have noticed I am getting the same issue where it will not full screen the video consistently.

**Comment by dvlden** [comment]
Hello @StealthyApple,

I got the credentials from my sister, she lives in Germany. I had to use a VPN as well (very bad experience) 

Amazon Prime seems very confusing. There are trailers, some shows that I can't even play, but purchase series/movie (as far as I understood)

Maybe she did not have a real amazon prime video, maybe she has a demo of it or something, not sure. Although, last time I checked and released the update, it worked on Amazon Prime for everything except trailers and ads. Which is fine, the focus is on entire shows.

If something seems broken again, get in touch with me via `n.dvlden@gmail.com` and let me know what is wrong, how it happens and such. Screenshots or video recording will help much. 
Also, the credentials and then if I manage to solve it or not, I'll let you know, to change your PW.

Regards,
Nenad Novakovic.
